x-pack/plugin/eql/qa/multi-cluster-with-security/src/test/java/org/elasticsearch/xpack/eql/RemoteClusterTestUtils.java

```diff
@@ -15,11 +17,10 @@ public static String remoteClusterIndex(String indexName) {
     }
 
     public static String remoteClusterPattern(String pattern) {
-        StringBuilder sb = new StringBuilder();
+        StringJoiner sj = new StringJoiner(",");
         for (String index: pattern.split(",")) {
```

Comment:
For future reference `String.split` is expensive since it compiles a regex pattern - however it does have a shortcut (at least in the recent JVMs) where for single chars it does string iteration. For the former case better to use `Strings.commaDelimitedListToStringArray`

Reply:
Thanks.